<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-8TTRVRQ" failures="1" tests="3" name="StandAlone error validations" time="36.308" errors="0" timestamp="2023-05-27T18:16:43 IST">
  <testcase classname="rahulshettyacademy.Tests.StandAlonePOM2ErrorValidation" name="loginErrorValidations" time="1.379">
    <skipped/>
  </testcase> <!-- loginErrorValidations -->
  <testcase classname="rahulshettyacademy.Tests.StandAlonePOM2ErrorValidation" name="loginErrorValidations" time="1.385">
    <failure type="java.lang.AssertionError" message="expected [Login Successfully] but found [Incorrect email or password.]">
      <![CDATA[java.lang.AssertionError: expected [Login Successfully] but found [Incorrect email or password.]
at rahulshettyacademy.Tests.StandAlonePOM2ErrorValidation.loginErrorValidations(StandAlonePOM2ErrorValidation.java:38)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 23 stack frames]]>
    </failure>
  </testcase> <!-- loginErrorValidations -->
  <testcase classname="rahulshettyacademy.Tests.StandAlonePOM2ErrorValidation" name="productErrorValidations" time="12.24"/>
</testsuite> <!-- StandAlone error validations -->
